packages/patterns/chatbot-list-view.tsx

```diff
@@ -219,44 +235,40 @@ export default recipe<Input, Output>(
             </aside>
 
             <aside slot="right">
-              {derive(selectedCharm.charm, (selected) => {
-                if (selected) {
-                  return (
-                    <>
-                      <div>
-                        <label>Backlinks</label>
-                        <ct-vstack>
-                          {selected?.backlinks?.map((
-                            charm: MentionableCharm,
-                          ) => (
-                            <ct-button
-                              onClick={handleCharmLinkClicked({ charm })}
-                            >
-                              {charm[NAME]}
-                            </ct-button>
-                          ))}
-                        </ct-vstack>
-                      </div>
-                      <details>
-                        <summary>Mentioned Charms</summary>
-                        <ct-vstack>
-                          {selected?.mentioned?.map((
-                            charm: MentionableCharm,
-                          ) => (
-                            <ct-button
-                              onClick={handleCharmLinkClicked({ charm })}
-                            >
-                              {charm[NAME]}
-                            </ct-button>
-                          ))}
-                        </ct-vstack>
-                      </details>
-                    </>
-                  );
-                } else {
-                  return null;
-                }
-              })}
+              {ifElse(
```

Comment:
btw, since this is in JSX, i think you could write that as success ? .. : ... instead.

and I think we also support the `{selected && <>...</>}` syntax, but right now it'll turn the second part into a derive, whereas ifElse treats the options as recipe-like inputs with lower taint. not a big difference, but we could change things so that `<condition> && <expression>` gets translated to `ifElse(condition, expression, null)` @mathpirate